core/.js/src/test/scala/io/udash/bindings/inputs/InputTest.scala

```diff
@@ -162,4 +173,77 @@ class InputTest extends AsyncUdashFrontendTest {
       r2.value should be("asd")
     }
   }
+
+  "run callback on state changes" in {
+    val p = Property[String]("ABC")
+    val result = Property("")
+    val input = TextInput(p, 0 millis, result.set(_))()
+    val inputEl = input.render
+
+    inputEl.value = "ABCD"
+    inputEl.onchange(new Event("change"))
```

Comment:
Previous tests were sending out `null` - is it not enough anymore?

Bonus points for finding some better way of testing input change. If not possible, please extract changing input value to some local extension method.

Reply:
It is, but it's clearer to me than sending null

Reply:
Cool, but still please add some local extension method or even just a private method to denoise this code.